.packaging/debian/docs/dial_tone.1

```diff
@@ -0,0 +1,15 @@
+.TH DIAL_TONE "1" "May 2012" "DIAL_TONE 3.6.0" "User Commands"
```

Comment:
This (and a bunch of others) are also in `docs/man`. Maybe we can do something clever to not duplicate those?

Reply:
Yes, seems like there is a set of man files already in tree I was not aware of.